code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -330,6 +330,32 @@
 /mob/living/silicon/ai/cancel_camera()
 	view_core()
 
+GLOBAL_LIST_EMPTY(empty_playable_ai_cores)
+
+/proc/roundstart_spawn_empty_ai_if_needed()
+	for(var/obj/effect/landmark/start/S in GLOB.landmarks_list)
+		if(S.name != "AI")
+			continue
+		if(locate(/mob/living) in S.loc)

```

Comment:
Should also check for pre-existing AI cores.

Reply:
isn't it already doing that?